indexing/src/main/resources/org/allenai/common/indexing/indexing.conf

```diff
@@ -39,16 +49,35 @@ org.allenai.common.indexing.base: {
       }
     }
   }
+}
 
-  buildIndexOptions: {
-    buildFromScratch: true
-    dumpFolder: "common/src/main/resources/org/allenai/ari/indexing/dump"
+org.allenai.common.indexing.paragraph_base: ${org.allenai.common.indexing.base} {
+  elasticSearch: {
+    indexType: "paragraph"
+    mapping: {
+      paragraph: {
+        dynamic: false,
+        properties: {
+          text: {
+            type: "multi_field",
+            fields: {
+              // Not sure yet what this "analyzer" field means.
```

Comment:
I used a browser to search my local elasticsearch index (https://github.com/mobz/elasticsearch-head), and using capital letters made all the results disappear.  This would be why?